fpm/alpine/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM php:5.6-fpm-alpine
+
+# install the PHP extensions we need
+RUN apk add --no-cache --virtual .build-deps \
+		autoconf gcc libc-dev make \
+		libpng-dev libjpeg-turbo-dev \
+	&& docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr \
+	&& docker-php-ext-install gd mysqli opcache \
+	&& find /usr/local/lib/php/extensions -name '*.a' -delete \
+	&& find /usr/local/lib/php/extensions -name '*.so' -exec strip --strip-all '{}' \; \
```

Comment:
Given that PHP's own `Dockerfile` strips all the executables it installs, I wonder if it wouldn't be more appropriate for `docker-php-ext-install` to do so as well (certainly would be more appropriate IMO to do this there generally than to do so here as a special case).